### PR TITLE
Bootargs tzdram base v1

### PR DESCRIPTION
**Pull request by vwadekar**
* Tegra: retrieve BL32's bootargs from bl32_ep_info
* Tegra132: set TZDRAM_BASE to 0xF5C00000

### CODE REVIEW

**Comment by danh-arm** [comment]
Can one of the admins verify this patch?

File: docs/plat/nvidia-tegra.md [hunk 1]
```diff
@@ -59,6 +59,9 @@ Preparing the BL31 image to run on Tegra SoCs
 'CROSS_COMPILE=<path-to-aarch64-gcc>/bin/aarch64-none-elf- make PLAT=tegra \
 TARGET_SOC=<target-soc e.g. t210|t132> SPD=<dispatcher e.g. tlkd> all'
```
Comment (danh-arm) [inline docs/plat/nvidia-tegra.md]: I've just noticed, shouldn't this command end with the `bl31` target instead of `all`?

**Comment by danh-arm** [comment]
test this please

File: docs/plat/nvidia-tegra.md (hunk 1 above)
Comment (vwadekar) [inline docs/plat/nvidia-tegra.md]: Yup. Agree.

**Comment by vwadekar** [comment]
New PR - #348

**Pull request closed without merging**